src/utils/VerificationValidationParser.cpp

```diff
@@ -153,9 +156,62 @@ void Parser::searchFinalDefense(Result* r) {
     }
 }
 
-Result* Parser::lc(const QString* terminalOutput) {
-    return nullptr; // TODO: implement
+Result* Parser::lc(const QString* cmd, const QString* terminalOutput) {
+	Result final;
+	final.terminalOutput = *terminalOutput;
+	final.cmd = *cmd;
```

Comment:
this shouldn't be necessary afaik; currently no references other than this assignment